requirements-app.txt

```diff
@@ -9,7 +9,7 @@ git+https://github.com/alphagov/digitalmarketplace-utils.git@46.1.2#egg=digitalm
 git+https://github.com/alphagov/digitalmarketplace-apiclient.git@19.12.0#egg=digitalmarketplace-apiclient==19.12.0
 git+https://github.com/alphagov/digitalmarketplace-content-loader.git@5.1.0#egg=digitalmarketplace-content-loader==5.1.0
 
-awscli==1.16.116
+awscli==1.16.121
```

Comment:
This going to bump `botocore` past our 1.11 pinning, which I think will cause problems...